sdk/src/test/java/com/horizen/account/utils/ChaChaPrngSecureRandomTest.java

```diff
@@ -0,0 +1,67 @@
+package com.horizen.account.utils;
```

Comment:
move this test to  `package com.horizen.utils` as well

Reply:
Done